### DISCUSSION TITLE
Deceptive site ahead : Shopify App Serve

### DISCUSSION THREAD

**Discussion by dani-sanomads** [Shopify/shopify-app-template-node#840] [Q&A] [answered]
Hi, 

Whenever I do any change in the code during 'shopify app serve', I get this display when I open the app. Do Let me know about it that why it is appearing and how can I solve it.

![image](https://user-images.githubusercontent.com/101345208/168977525-f6da2b57-ddc6-4972-b0f7-b50a92feeaa8.png)

**Comment by mirceapiturca** [comment]
I am getting the same Google Chrome warning. This is most likely caused by the automatic top-level redirect.
I am concerned that this may appear also in production. You can bypass it by clicking on "Details".

Can anyone confirm if this can happen in production?
I like this type of auth redirect but this would be a deal breaker.

**Comment by dani-sanomads** [comment]
Thanks. Its working fine on the Heroku after deployment but if anyone can mention the exact problem then we can overcome it as well.

**Comment by TheSecurityDev** [comment, marked as answer]
It's because you're using ngrok for development. Ngrok is used a lot by malicious sites, so Google Chrome flags it sometimes. Just ignore the warnings, and don't use ngrok for production.

**Comment by dani-sanomads** [comment]
Okay, I got it . Thanks.
